libs/documentation/src/lib/components/collapse/demos/basic/collapse-basic.component.html

```diff
@@ -1,11 +1,11 @@
 <div class="text-center">
   <button type="button" class="usa-button text-center" [attr.aria-expanded]="!isCollapsedContent"
-    aria-controls="collapseID" (click)="isCollapsedContent = !isCollapsedContent"> Show / Hide Content
+    aria-controls="collapseID" (click)="isCollapsedContent = !isCollapsedContent" (click)="onClick.emit($event)"> Show / Hide Content
   </button>
 </div>
 
 <span aria-live="polite">
-  <p id="collapseID" [sdsCollapse]="isCollapsedContent" class="bg-base-lighter margin-top-1 padding-2">
+  <p id="collapseID" [sdsCollapse]="onClick.emit()" class="bg-base-lighter margin-top-1 padding-2" [ngClass]="classes">
```

Comment:
Why is an event being passed into this input? That input looks like it was handling a boolean before.